random-beans/src/main/java/io/github/benas/randombeans/util/Constants.java

```diff
@@ -34,37 +34,34 @@
  *
  * @author Mahmoud Ben Hassine (mahmoud.benhassine@icloud.com)
  */
-public abstract class Constants {
```

Comment:
+1 . Same here. I think it would be better to keep it a class rather than an interface